stable/minio/templates/minio_secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-user
```

Comment:
This name and the name declared in the deployment (line 34) do not line up. This causes the pod to fail to start.